addons/sale_timesheet/static/src/components/so_line_field/so_line_field.js

```diff
@@ -13,7 +13,7 @@ export class SoLineField extends Many2OneField {
             update(value, params);
             if ( // field is unset AND the old & new so_lines are different
                 !this.props.record.data.is_so_line_edited &&
-                this.props.value[0] != (value[0] && value[0].id)
+                this.value != value
```

Comment:
Is this really good? 

I am wondering if value is not an array containing 2 elements (the `id` and the `display_name`)

Reply:
Hello @xavierbol 
Done :+1: